scripts/windows/download.bat

```diff
@@ -1,2 +1,69 @@
-curl -O -L https://github.com/codex-storage/nim-codex/releases/download/v0.1.0/codex-v0.1.0-windows-amd64.zip
-unzip codex-v0.1.0-windows-amd64.zip
+@echo off
+setlocal enabledelayedexpansion
+
+call utils.bat
+
+:: Set variables
+set "OS=windows"
+call :get_arch ARCH
+set "ARCHIVE_EXT=.zip"
+set "EXE_EXT=.exe"
+set "VERSION=v0.1.0"
+set "BASE_URL=https://github.com/codex-storage/nim-codex/releases/download/%VERSION%"
+set "EXTRACT_DIR=.\"
+set "BINARY_NAMES=codex"
+
+:: Download, verify, and extract each binary
```

Comment:
Does not properly verify checksum of binary and extract from .zip file.

Testing it on W11 machine in Powershell (not UCRT or MSYS) and ran into checksum issue in download.bat.

![image](https://media.discordapp.net/attachments/1255090454798794802/1255100991347490888/image.png?ex=667be762&is=667a95e2&hm=233adba4fadd41bcee80581da731f3df441b84dd3a364b14f56b2afae87fc03d&=&format=webp&quality=lossless&width=1659&height=934)

Failed on executing Codex binary (didn't extract zip). 

![image](https://media.discordapp.net/attachments/1255090454798794802/1255101225670803557/image.png?ex=667be79a&is=667a961a&hm=daec98495b66c809f9274ac05aa766389d0fbfbf41a39d47980aec8e68167b8e&=&format=webp&quality=lossless&width=1659&height=934)

Reply:
These scripts were auto-ported by Claude to native windows and are not fully functional yet. The first error with shasum checking is a case-senstive check which should be easy to fix. We haven't had a chance to get to the rest. Any and all help on Windows is appreciated. For now, we're gonna leave them as is and work with the assumption that everyone on windows has a *nix-like environment on their machine, and we can update these scripts in the coming days